update: added missed step about adding dependency to the web socket s… (#60)
JetBrains/kotlin-mobile-docs: topics/connect-to-platform-specific-apis.md

}
```

> Android implementation uses [OkHttp](https://square.github.io/okhttp/) third-party library, so you need to declare the corresponding dependency in the shared module `build.gradle`:

Reviewer: ```suggestion
Android implementation uses the third-party library [OkHttp](https://square.github.io/okhttp/). Add the corresponding dependency to `build.gradle(.kts)` in the shared module:
```

> ```kotlin

Reviewer: Please provide a Groovy sample as well.

sourceSets {
    val androidMain by getting {
        dependencies {

>             implementation("com.squareup.okhttp3:okhttp:4.9.0")

Reviewer: Is it OK to use specific version here? Will it work in the future?

Author: Fixed

        }
    }
}
```

> iOS implementation uses `NSURLSession` from the standard Apple SDK, so you don't need to add any other dependencies. 

Reviewer: ```suggestion
iOS implementation uses `NSURLSession` from the standard Apple SDK and doesn't require additional dependencies. 
```
